tools/scylla-nodetool.cc

```diff
@@ -216,9 +216,7 @@ void compact_operation(scylla_rest_client& client, const bpo::variables_map& vm)
         }
         client.post(format("/storage_service/keyspace_compaction/{}", keyspace), std::move(params));
     } else {
-        for (const auto& keyspace : get_keyspaces(client)) {
-            client.post(format("/storage_service/keyspace_compaction/{}", keyspace), std::move(params));
-        }
+        client.post("/storage_service/compact", std::move(params));
```

Comment:
There is no params here. In fact, we moved an empty params map on each iteration above, which is a bug, albeit a harmless one.

Reply:
Hmm, please check again.
1. params may have a `sf` flag.
2. which iteration above?  This is in deleted lines of code...

Reply:
Yes, I meant the deleted code. I missed that you added the `sf` flag, then indeed this is right (passing empty map is also technically correct anyway).

Reply:
fixed the loop in next version (the comment is valid for `api: force_keyspace_compaction: add skip_flush option` patch)